students/tonylee/test_mailroom_part4.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import os.path
+
+from mailroom_part4 import donation_history, thank_you_message, send_letter
+
+class TestMailroomPart4(unittest.TestCase):
+    def test_all_donor_names(self):
+        self.assertEqual(['Tony Lee', 'Michelle Cao', 'Andy Arko', 'Tom Ludwinski'], list(donation_history.keys()))
```

Comment:
this test is testing python functionality (like contents of keys)

your unit tests should test your code logic.